### PR TITLE
Add WANT TO CONTRIBUTE with github repo link on contributors page

### LINKED ISSUE #54: Add "Want to contribute" with github repo link on contributors page
In the `contributors.html` page, add something, so that if someone wants to contribute can find the Github repo and can contribute or report an issue.

### PR DESCRIPTION
**Pull request by MayaSatishRao**
Added a new section which contains a anchor tag and a h2 element. The anchor tag links to the github repository. 
Fixes #54

### CODE REVIEW

**Comment by MayaSatishRao** [comment]
@Signior-X  This is my first open-source contribution. Please do tell me whether the changes that I have made meets your requirement?

File: contributors.html [hunk 1]
```diff
@@ -81,6 +81,18 @@ <h2 class="text-center">Current Maintainer - Signior-X</h2>
       </div>
     </section>
 
+   <!-- Contributing button -->
+    <section class="probootstrap-section">
+     <div class=" text-center">
+       <h2>Want to contribute to our repository?</h2>
```
Comment (Signior-X) [inline contributors.html:87]: Change it to `Want to contribute` from `Want to contribute to our repository`.

File: css/style.css [hunk 2]
```diff
@@ -3146,4 +3146,13 @@ a:hover, a:focus, a:active {
...
-}
\ No newline at end of file
+}
+
+.contributing-button {
+  width:25%;
+  font-size: 20px;
+  align-items: center;
+  position:relative;
+  left:38%;
```
Comment (Signior-X) [inline css/style.css:3156]: Instead of `left:38%`, use `margin: auto`.
This will automatically center the button.

**Review by Signior-X — APPROVED** [review]
Great work @MayaSatishRao 
Yes, it's as per expectation. I just made few comments and changes as shown here.

Merging this one. Thanks!

**Comment by Signior-X** [comment]
Merged :tada:

**Comment by MayaSatishRao** [comment]
Thanks for such quick response. I am very happy that I could contribute
something to this project.
  Thanks a lot

**Comment by Godwindaniel10** [comment]
@MayaSatishRao , awesome job🎌,  but it will be great to make the button responsive and fit ...

make button responsive

**Pull request merged**